### ISSUE TITLE
Thank you for your help

### ISSUE THREAD

**Issue by lindermanfire** [j-alex-hanson/speedy-splat#3]
Thanks for your work! But When I use it,it errors:
def get_state_dict(net_type: str = 'alex', version: str = '0.1'):
    # build url
    url = 'https://raw.githubusercontent.com/richzhang/PerceptualSimilarity/' \
        + f'master/lpips/weights/v{version}/{net_type}.pth'
 # download
    old_state_dict = torch.hub.load_state_dict_from_url
can not load it

**Comment by j-alex-hanson** [comment]
It looks like the issue is with downloading the model weights for the LPIPS metric.

A couple of things to check:
-  **Storage space:** Ensure that you have enough space in your `$HOME/.cache` directory, as PyTorch downloads model weights there by default.
- **Internet access:** Check if the device running the code has internet access. If not, you may need to manually download the model weights before running the code.

This code comes from the original INRIA 3D-GS implementation, specifically their LPIPS evaluation. I tested it on my device and was able to download the model weights successfully.

**Comment by lindermanfire** [comment]
Thanks! I test in windows,it's ok,but in linux ,it errors,do you know can I use "torch.hub.load_state_dict_from_url" to load local vgg16 model or use other function to local localmodel?

**Comment by lindermanfire** [comment]
> It looks like the issue is with downloading the model weights for the LPIPS metric.
> 
> A couple of things to check:
> 
>     * **Storage space:** Ensure that you have enough space in your `$HOME/.cache` directory, as PyTorch downloads model weights there by default.
> 
>     * **Internet access:** Check if the device running the code has internet access. If not, you may need to manually download the model weights before running the code.
> 
> 
> This code comes from the original INRIA 3D-GS implementation, specifically their LPIPS evaluation. I tested it on my device and was able to download the model weights successfully.

Thank you,I solve it
